go/vt/vtctl/reparentutil/emergency_reparenter.go

```diff
@@ -420,9 +421,11 @@ func (erp *EmergencyReparenter) findMostAdvanced(
 	winningPosition := tabletPositions[0]
 
 	// We have already removed the tablets with errant GTIDs before calling this function. At this point our winning position must be a
-	// superset of all the other valid positions. If that is not the case, then we have a split brain scenario, and we should cancel the ERS
+	// superset of all the other valid positions - unless the tablet with that position is taking a backup, in which case it might
+	// have a larger GTID set but we are still not choosing it. If that is not the case, then we have a split brain scenario, and
```

Comment:
```suggestion
	// have a larger GTID set but we are still not choosing it. This can lead to split brain, so
```

Reply:
I had a hard time understanding the previous iteration, I hope this is easier to read.

Reply:
Fixed.